models/src_mailchimp.yml

```diff
@@ -2,13 +2,572 @@ version: 2
 
 sources:
   - name: mailchimp
-    schema: "{{ var('schema', 'mailchimp') }}"
+    schema: "{{ var('mailchimp_schema', 'mailchimp') }}"
     database: "{{ var('database', target.database) }}"
 
+    loader: fivetran
+    loaded_at_field: _fivetran_synced
```

Comment:
It looks like for the relevant tables the `_fivetran_synced` and `_fivetran_deleted`(where applicable) do not have field definitions. Would you be able to add those in.